scripts/remap-coverage.js

```diff
@@ -15,10 +15,7 @@ const remapIstanbulExecutable = path.join(
 );
 
 const coverageParentFolder = path.join(
-  __dirname,
```

Comment:
I'm assuming this is because `remap-coverage` is being used across the project and not exclusively on system tests ?

Reply:
Yeah, the `coverage` script (added to packages previously missing it) uses `remap-coverage`. Once we get the other components working, we'll have to use `remap-coverage` across the project.